src/invoices/InvoiceDetails/InvoiceActions/InvoiceActions.js

```diff
@@ -117,6 +119,19 @@ const InvoiceActions = ({
           </Icon>
         </Button>
       )}
+      {isCancelable && (
+        <IfPermission perm="invoice.invoices.item.put">
```

Comment:
hm, why do you check put permission? user with edit permission and without cancel will be able to cancel in this case

Reply:
update, check please